templates/common/infra/bicep/core/ai/cognitiveservices.bicep

```diff
@@ -1,3 +1,4 @@
+metadata description = 'Creates an Azure Cognitive Services instance.'
```

Comment:
@savannahostrowski - I think we should use the word "service" instead of instance.  I don't typically here people referring to a service as an instance.  Do you?

Reply:
I'm not picky - I just think Azure Cognitive Services service reads poorly.

Reply:
@savannahostrowski, @jongio Can we consider updating to `Creates an Azure Cognitive Services Account`, or `Creates an Azure Cognitive Services`?

Reply:
we can leave as-is